source/includes/dont-use-api-key-in-browsers.rst

```diff
@@ -0,0 +1,6 @@
+.. important::
+   
+   If you're authenticating from a browser or another user-facing environment,
```

Comment:
```suggestion
   If you're authenticating from a browser or another user-facing client application,
```

i think this captures the idea a little better, but if i'm missing the point feel free to reject suggestion